packages/manager/modules/email-domain/src/general-informations/email-domain-general-informations.controller.js

```diff
@@ -139,6 +139,20 @@ export default class EmailTabGeneralInformationsCtrl {
       });
   }
 
+  getSpfConfig() {
+    const spfValid = this.domain.isSPFValid;
+
+    if (spfValid === 'valid') {
+      return 'ok';
+    }
+
+    if (spfValid === 'unknown' || spfValid === 'checkFail') {
+      return 'fail';
+    }
+
+    return spfValid;
+  }
```

Comment:
I think it would be more efficient to end on a boolean

```suggestion
  hasSpfConfig() {
    return !['invalid', 'none'].includes(this.domain.isSPFValid);
  }
  
  isValidSpfConfig() {
    return this.domain.isSPFValid === 'valid';
  }
```